### DISCUSSION TITLE
Highlight documentation window colours

### DISCUSSION THREAD

**Discussion by megalithic** [hrsh7th/nvim-cmp#1124] [Ideas]
Cross-posting from this plugin: https://github.com/brenoprata10/nvim-highlight-colors/issues/15

I'd love to know how to go about applying colour highlighting for the documentation window. Is it possible to push the documentation window content through a function that can apply an additional callback to do the colour highlighting transformations?

OR, even easier, can we apply a `buftype` to the documentation window?

<img src="https://user-images.githubusercontent.com/3678/183708262-5b4ad176-be9a-4e56-a4de-51ec85430665.png" />

**Comment by l00sed** [comment]
You can use the Colorizer plugin:
```lua
  -- Colorizer
  {
    'norcalli/nvim-colorizer.lua',
    config = function()
      require('colorizer').setup({
        '*',
      }, {
        RGB      = true,   -- #RGB hex codes
        RRGGBB   = true,   -- #RRGGBB hex codes
        names    = true,   -- "Name" codes like Blue
        RRGGBBAA = true,  -- #RRGGBBAA hex codes
        rgb_fn   = true,  -- CSS rgb() and rgba() functions
        hsl_fn   = true,  -- CSS hsl() and hsla() functions
        css      = true,  -- Enable all CSS features: rgb_fn, hsl_fn, names, RGB, RRGGBB
        css_fn   = true,  -- Enable all CSS *functions*: rgb_fn, hsl_fn
        -- Available modes: foreground, background
        mode     = 'background' -- Set the display mode.
      })
    end
  },
```
![image](https://user-images.githubusercontent.com/23065167/209212702-ead08ef1-2edf-44e2-91a7-3d67f9a3224e.png)

**Reply by megalithic** [reply]
this used to not work; i'll try it again.

**Reply by l00sed** [reply]
I think the key is setting `'*'` to make Colorizer run everywhere.
